### ISSUE TITLE
Issue with Laravel 10 (documentation is not generated)

### ISSUE THREAD

**Issue by AUnhurian** [RonasIT/laravel-swagger#78]
Hello guys,
I have an issue with working on this package in Laravel.

I changed my PHPUnit library version to 10.0, but without any success
<img width="449" alt="image" src="https://user-images.githubusercontent.com/25745274/221954534-0794a391-b4c8-4560-b6c4-f2fbd061ee4d.png">

It is really a misunderstanding for me because it not works.
I have a next test example:
```

use Illuminate\Foundation\Testing\RefreshDatabase;
use Symfony\Component\HttpFoundation\Response;
use Tests\TestCase;

class ExampleTest extends TestCase
{
    public function testBasicTest()
    {
        $response = $this->get('/api/test');

        $response->assertStatus(200);
    }

    public function testGetListNotFound() {
        $response = $this->json('get', '/api/test', [
            'not-found' => true
        ]);

        $response->assertStatus(Response::HTTP_NOT_FOUND);
    }

    public function testGetListNoAuth() {
        $response = $this->json('get', '/api/test', [
            'need-authorization' => true
        ]);

        $response->assertStatus(Response::HTTP_UNAUTHORIZED);
    }

    public function testGetListWrongParameters() {
        $response = $this->json('get', '/api/test', [
            'test-parameter' => 'test'
        ]);

        $response->assertStatus(Response::HTTP_UNPROCESSABLE_ENTITY);
    }
}
```

TestController:
<img width="545" alt="image" src="https://user-images.githubusercontent.com/25745274/221955298-335f7359-794a-42f0-b5bc-e834f914d488.png">

TestGetRequest:
```

use Illuminate\Foundation\Http\FormRequest;

/**
 * @description
 *  This request designed to demonstrate request with full annotation and response witch contain some data.
 * It has multi-line description witch will be displayed in Annotation Notes block of documentation.
 * It has custom summary, response code descriptions and parameters descriptions.
 *
 * @summary Test Get Request with full annotations
 *
 * @_204 This request has successfully done
 * @_401 You must remove need-authorization flag from input parameters for pass authorization.
 * @_404 We so sorry but you entity not exists.
 * @_422 Wrong input parameter. It must be integer
 *
 * @need-authorization If this parameter is true then you will get 401 response
 * @not-found If this parameter is true then you will get 404 response
 * @test-parameter This parameter designed for demonstrate unprocesable entity response
 */
class TestGetRequest extends FormRequest
{
    /**
     * Determine if the user is authorized to make this request.
     *
     * @return bool
     */
    public function authorize()
    {
        return true;
    }

    /**
     * Get the validation rules that apply to the request.
     *
     * @return array
     */
    public function rules()
    {
        return [
            'not-found' => 'boolean',
            'need-authorization' => 'boolean',
            'test-parameter' => 'integer'
        ];
    }
}
```

api route:
<img width="798" alt="image" src="https://user-images.githubusercontent.com/25745274/221955600-d9134086-fe81-484c-9db9-c8048e425dea.png">

auto-doc config:
```

return [

    /*
    |--------------------------------------------------------------------------
    | Documentation Route
    |--------------------------------------------------------------------------
    |
    | Route which will return documentation
    */
    'route' => '/swagger',

    /*
    |--------------------------------------------------------------------------
    | Global application prefix
    |--------------------------------------------------------------------------
    |
    | Usefully in case the webserver using a path to route requests to the app
    | In case your app available at https://some.domain.com/service - use
    | /service as global prefix config
    */
    'global_prefix' => env('SWAGGER_GLOBAL_PREFIX', ''),

    /*
    |--------------------------------------------------------------------------
    | Info block
    |--------------------------------------------------------------------------
    |
    | Information fields
    */
    'info' => [

        /*
        |--------------------------------------------------------------------------
        | Documentation Template
        |--------------------------------------------------------------------------
        |
        | You can use your custom documentation view
        */
        'description' => 'auto-doc::swagger-description',
        'version' => '0.0.0',
        'title' => 'Name of Your Application',
        'termsOfService' => '',
        'contact' => [
            'email' => 'your@email.com'
        ],
        'license' => [
            'name' => '',
            'url' => ''
        ]
    ],
    'swagger' => [
        'version' => '2.0'
    ],

    /*
    |--------------------------------------------------------------------------
    | Base API path
    |--------------------------------------------------------------------------
    |
    | Base path for API routes. If config is set, all routes which starts from
    | this value will be grouped.
    */
    'basePath' => '/',
    'schemes' => [],
    'definitions' => [],

    /*
    |--------------------------------------------------------------------------
    | Security Library
    |--------------------------------------------------------------------------
    |
    | Library name, which used to secure the project.
    | Available values: "jwt", "laravel", "null"
    */
    'security' => '',
    'defaults' => [

        /*
        |--------------------------------------------------------------------------
        | Default descriptions of code statuses
        |--------------------------------------------------------------------------
        */
        'code-descriptions' => [
            '200' => 'Operation successfully done',
            '204' => 'Operation successfully done',
            '404' => 'This entity not found'
        ]
    ],

    /*
    |--------------------------------------------------------------------------
    | Driver
    |--------------------------------------------------------------------------
    |
    | The name of driver, which will collect and save documentation
    | Feel free to use your own driver class which should be inherited from
    | `RonasIT\Support\AutoDoc\Interfaces\SwaggerDriverInterface` interface,
    | or one of our drivers from the `drivers` config:
    */
    'driver' => env('SWAGGER_DRIVER', 'storage'),

    'drivers' => [
        'local' => [
            'class' => LocalDriver::class,
            'production_path' => storage_path('documentation.json')
        ],
        'remote' => [
            'class' => RemoteDriver::class,
            'key' => env('SWAGGER_REMOTE_DRIVER_KEY', 'project_name'),
            'url' => env('SWAGGER_REMOTE_DRIVER_URL', 'https://example.com')
        ],
        'storage' => [
            'class' => StorageDriver::class,

            /*
            |--------------------------------------------------------------------------
            | Storage disk
            |--------------------------------------------------------------------------
            |
            | One of the filesystems.disks config value
            */
            'disk' => env('SWAGGER_STORAGE_DRIVER_DISK', 'public'),
            'production_path' => 'documentation.json'
        ]
    ],

    /*
    |--------------------------------------------------------------------------
    | Paths to additional documentation
    |--------------------------------------------------------------------------
    | An array of documentation paths to merge with the main documentation.
    |
    | For example, if your additional documentation is located in storage/additional_docs/example.json
    | you need to add 'storage/additional_docs/example.json' to additional_paths, if your additional
    | documentation is located in the root directory of your project you need to add the filename to additional_paths
    */
    'additional_paths' => [],

    /*
    |--------------------------------------------------------------------------
    | Response example array items limit
    |--------------------------------------------------------------------------
    | All array responses will be automatically cut for the config items count
    |
    | Note: you should collect documentation after the config change
    */
    'response_example_limit_count' => 5,

    /*
    |--------------------------------------------------------------------------
    | Swagger documentation visibility environments list
    |--------------------------------------------------------------------------
    |
    | The list of environments in which auto documentation will be displaying
    */
    'display_environments' => [
        'local',
        'testing',
        'development'
    ],

    'config_version' => '2.4'
];
```

But I have still received not whole json in the documentation.json file
<img width="939" alt="image" src="https://user-images.githubusercontent.com/25745274/221955994-7734c008-1c44-4617-9446-0a034927d101.png">

Could you please help me?

**Comment by ELWAHAB** [comment]
I have checked data before and after saving in the SwaggerService by this code
<img width="880" alt="image" src="https://user-images.githubusercontent.com/25745274/221957082-07f960ab-9af5-4693-a2e2-cd9f5b4653c4.png">

And it is really weird because it looks like the data in the StorageDriver is not saving
<img width="972" alt="image" src="https://user-images.githubusercontent.com/25745274/221957361-5c39f021-6593-4e1f-97b4-996bf35165b4.png">

<img width="666" alt="image" src="https://user-images.githubusercontent.com/25745274/221957468-7307823c-e58d-47b5-a6cc-5fe1f691d6ad.png">

I also launched `php artisan swagger:push-documentation` but it is not helped me generate documentation for `'/api/test'` endpoint

**Comment by DenTray** [comment]
@ELWAHAB thanks for your feedback, we'll investigate this one at the nearest time!

**Comment by Ferikl** [comment]
maybe issue with phpunit v10

**Comment by DenTray** [comment]
@AUnhurian we've made a fix to this one, so you can test it on the `dev-master` version

**Comment by donjenflo** [comment]
I have a similar problem. Looking forward to the new build.

**Comment by DenTray** [comment]
@donjenflo thanks for your feedback, are you trying to install `dev-master` version?

**Comment by donjenflo** [comment]
@DenTray  I tried now and it worked!  thx its nice software!

**Comment by DenTray** [comment]
@donjenflo thanks! We expect next release in coming month, so you'll be able to use stable version. Please follow to this task.

**Comment by DenTray** [comment]
Released in [2.2 version](https://github.com/RonasIT/laravel-swagger/releases/tag/2.2)
